src/xDynamicSimulation/cDynamicSimulation/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+#========================================================================
+# Author: Johannes Mayer
+# Date: 15.03.2018
+#=========================================================================
+
+add_definitions(-DBOOST_ALL_NO_LIB)
+
+if (SIRF_INSTALL_DEPENDENCIES AND WIN32)
+    set(Boost_DLL_DIR ${Boost_LIBRARY_DIR_RELEASE})
+    message(STATUS "Install boost shared libraries from ${Boost_DLL_DIR} ")
+	foreach (__boost_lib system filesystem thread date_time chrono)
+	   file(GLOB Boost_DLL "${Boost_DLL_DIR}/boost_${__boost_lib}*.dll")
+       install( FILES ${Boost_DLL} DESTINATION bin )
+	endforeach()
+  endif()
+
+include_directories(${PROJECT_SOURCE_DIR}/src/common/include)
+include_directories(${PROJECT_SOURCE_DIR}/src/xGadgetron/cGadgetron)
+include_directories(${PROJECT_SOURCE_DIR}/src/xSTIR/cSTIR)
+include_directories(${PROJECT_SOURCE_DIR}/src/Registration/Reg)
+include_directories(${PROJECT_SOURCE_DIR}/src/iUtilities)
+include_directories("${GADGETRON_INCLUDE_DIR}/../")
```

Comment:
probably shouldn't be here as should be taken care of by `target_link_libraries`